Update README.md (#7)
usegalaxy-eu/branding: README.md

## Galaxy Europe

> - Use this anywhere we're really talking about the European Galaxy project as a collective, e.g. posters next to your ELIXIR/de.NBI logos.

Reviewer: I would not mention this. Most people do not know what de.NBI is and maybe even ELIXIR outside of Europe.

> - Use this anywhere we're really talking about the European Galaxy project as a collective, e.g. posters next to your ELIXIR/de.NBI logos.

Author: ```suggestion
- Use this anywhere you refer to the European Galaxy project/community.
```

> - Do not use this when referring to specific sub-teams fo the Galaxy Project Europe.

Author: ```suggestion
- Do not use this when referring to specific sub-teams of the Galaxy Project Europe.
```

## Freiburg Galaxy

> - Use this anywhere we're really talking about the Freiburg Galaxy Team, e.g. any articles/trainings produced by Freiburg Galaxy Team.

Author: ```suggestion
- Use this anywhere you refer to the Freiburg Galaxy Team, e.g. any articles/training activities produced by the Freiburg Galaxy Team.
```

> - Do not use this when referring to the collective team.

Author: ```suggestion
- Do not use this when referring to a broader community.
```

## useGalaxy.eu

> - Use this anywhere where you want to direct users to our server, e.g. trainings where you want to remind users "here is the URL" and stickers you

Author: ```suggestion
- Use this anywhere where you want to direct users to the European Galaxy server, e.g. trainings where you want to remind users "here is the URL" and stickers you
```
